docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -60,13 +60,13 @@ All Camunda 8 Run components can be accessed using the username/password combina
 
 Tasklist and Operate are available at:
 
-- Tasklist: `http://localhost:8080/tasklist`
-- Operate: `http://localhost:8080/operate`
+- Tasklist: [http://localhost:8080/tasklist](http://localhost:8080/tasklist)
+- Operate: [http://localhost:8080/operate](http://localhost:8080/operate)
 
 The following components do not have a web interface, but the URLs may be required for additional configuration:
 
-- Zeebe Gateway: `http://localhost:26500`
-- Connectors: `http://localhost:8085`
+- Zeebe Gateway: [http://localhost:26500](http://localhost:26500)
+- Connectors: [http://localhost:8085](http://localhost:8085)
```

Comment:
I don't think that localhost references should be direct links. We don't use them as direct links anywhere in the docs, and they'll only be functional for someone who has Camunda actively running in this way - in actuality, a user here is more likely to get their automatic Operate window, and be provided these links in the C8Run output. This should be considered more of a reference, not a suggestion to Go There.